releasenotes/AzureDevOpsPipelineExtension.md

```diff
@@ -0,0 +1,15 @@
+## Azure App Configuration Extension release notes
+
+The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version is printed out to console in the step of running the Azure App Configuration extension: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
+
+### 1.6.3
+* Fix case insensitive for keys in the Azure App Configuration extension side.
+* Support auto-populated `App Configuration name` dropdown from a textbox.
+* Fix framework forcing to print out any key-values when failed to set as environment variables; the extension only prints key with a mask value `****` if process failed to set as environment variable.
```

Comment:
I'm a little confused at what this is trying to say. It might help to say it like this:

* Attempting to set environment variables with invalid _keys_ now prints a masked value, `****`, instead of the actual value of the key-value.

Reply:
Yes, will update it to this.